peak/mapper/mapper.py

```diff
@@ -31,7 +31,8 @@ def _convert_io_types(peak_io):
 
 def gen_mapping(
         peak_class : Peak,
-        isa : tp.Type[ISABuilder],
+        bv_isa : tp.Type[BitVector],
```

Comment:
These are not right type annotations right? They're still `ISABuilder` objects.

Reply:
Fixed